src/components/TopBar.tsx

```diff
@@ -243,10 +245,13 @@ const TopBar = ({ togglePanel }: TopBarProps) => {
                 <div className="flex flex-col px-2 justify-center">
                   <span className="font-bold">{user?.name}</span>
                   {userAgencyResult?.UserLatestAgency[0]?.agency && (
-                    <span>
+                    <p className={"text-sm"}>
                       {`Zenika ${userAgencyResult?.UserLatestAgency[0]?.agency}`}
-                    </span>
+                    </p>
                   )}
+                  <p className={"text-xs"}>{`${t(
+                    "myProfile.lastSeen"
+                  )} : ${lastSeen.toLocaleString("fr-FR")}`}</p>
```

Comment:
are the singapore users using Skillz ? aha

Reply:
Good luck to fix this then aha, sorry Benjamin ! But yes, this should adapt to where we use Skillz 😬

Reply:
It is now fixed :))